.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Release checklist
+about: Prepare for a new release version of scpcaTools
+title: Prepare for scpcaTools release vX.X.X
+labels: release
+assignees: ''
+
+---
+
+## Steps for a new release of `scpcaTools`
+
+### Preparing for the release and testing
+
+- [ ] Are all of the issues planned for this release resolved? If there are any issues that are unresolved, mark this issue as blocked by those on ZenHub.
+- [ ] Did all automated tests and the Docker build in this repo pass in CI? (You may have to wait for the docker image to build. Check [Github actions](https://github.com/AlexsLemonade/scpcaTools/actions/workflows/build-docker.yaml) before proceeding to the next step
```

Comment:
```suggestion
- [ ] Did all automated tests and the Docker build in this repo pass in CI? (You may have to wait for the docker image to build. Check [Github actions](https://github.com/AlexsLemonade/scpcaTools/actions/workflows/build-docker.yaml) before proceeding to the next step)
```

Or maybe the `)` goes after "wait for the docker image to build."

Reply:
Just made it not a parenthetical.